index.html

```diff
@@ -9,9 +9,29 @@
 
     <link rel="stylesheet" href="styles.css" />
 
+    <script src="utlities.js" defer></script>
     <script src="app.js" defer></script>
+    
   </head>
   <body>
-      <h1>Surprise</h1>
+    <h1>Surprise</h1>
+
+    <form id="choose-ingredient-form">
+      <h2>Choose your favorite Ingredient</h2>
+      <input type="radio" id="strawberry" name="ingredient" value="strawberry"
+        checked/>
+      <!-- "checked" is added so when the page loads it has strawberry already selected -->
+      <label for="strawberry">Strawberry</label><br />
+      <input type="radio" id="chocolate" name="ingredient" value="chocolate" />
```

Comment:
I would name it id="chocolateRadio"

Reply:
I remove the ids, the code is not really using them :)